docs/quick_start.md

```diff
@@ -0,0 +1,182 @@
+## Quick Start for OpenFeature Operator
```

Comment:
I think this is a really nice addition. Using busy-box, having no ingress, etc, is nice b/c it's very "barebones".

Reply:
Agree, we should keep it minimal and let users get things up and running fast. Later, we can refer them to detailed guides